CONTRIBUTING.md

```diff
@@ -0,0 +1,245 @@
+# How to contribute to OMZ
+
+From this document you will know how to contribute your model to OpenVINO&trade; Open Model Zoo. Almost any model from supported frameworks (see list below) can be added. It could be done in few simple steps.
+
+1. [Model location](#model-location)
+2. [Model conversion](#model-conversion)
+3. [Demo](#demo)
+4. [Accuracy validation](#accuracy-validation)
+7. [Configuration file](#configuration-file)
+6. [Documentation](#documentation)
+7. [Pull request requirements](#pull-request-requirements)
+
+List of supported frameworks:
+* Caffe\*
+* Caffe2\* (by conversion to ONNX\*)
+* TensorFlow\*
+* MXNet\*
+* PyTorch\* (by conversion to ONNX\*)
+
+## Model location
+
+Upload your model to any Internet file storage with easy and direct access to it. It can be www.github.com, GoogleDrive\*, or any other.
```

Comment:
It's not clear what "easy and direct access" means. In fact, I would argue that access to Google Drive is neither easy nor direct.

The real requirements are that the model must either be downloadable from a direct HTTP(S) link or from Google Drive.

(Note, BTW, that Google Drive is two words, not one.)